cypress/integration/tests/administrator/proxy/proxy.enable.disable.test.ts

```diff
@@ -22,6 +30,35 @@ describe("Proxy operations", () => {
         preservecookies();
     });
 
+    it("Http Proxy port and host field validation", function () {
+        Proxy.open();
+        selectCheckBox(ProxyViewSelectors.httpSwitch);
```

Comment:
Is it same as enableProxy method ?

Reply:
Solved in bdcd68008f4c29413d50f2ffabf64a035a98d55d.